doc/core_api.md

```diff
@@ -19,25 +18,43 @@ Notice that cartodb.SQL is also included in that JavaScript file
 
 Fetch the tile template for the layer definition.
 
-### Arguments
+#### Arguments
 
-+ **layerOptions**: the data that defines the layer. It should contain at least user_name and sublayer list. These are the available options:
+Arguments | Description
+--- | ---
+layerOptions | the data that defines the layer. It should contain at least `user_name` and `sublayers` list.
+
+options | 
```

Comment:
The argument is actually called `layerOptions` in the method definition (`cartodb.Tiles.getTiles(layerOptions, callback)`). Would it make sense to do something like: 

![screen shot 2015-11-04 at 12 10 01](https://cloud.githubusercontent.com/assets/390398/10936443/021652c0-82ed-11e5-81b0-4792711c568b.png)